src/config/frontend/EdFi.DmsConfigurationService.Frontend.AspNetCore.Tests.Unit/Modules/IdentityModuleTests.cs

```diff
@@ -62,13 +61,14 @@ public async Task Given_valid_client_details()
         using var client = factory.CreateClient();
 
         // Act
-        var requestContent = new
-        {
-            clientid = "CSClient1",
-            clientsecret = "test123@Puiu",
-            displayname = "CSClient1",
-        };
-        var response = await client.PostAsJsonAsync("/connect/register", requestContent);
+        var requestContent = new FormUrlEncodedContent(
+        [
+              new KeyValuePair<string, string>("clientid", "CSClient1"),
```

Comment:
Can you try formatting this file? I'm showing some format complaints when I pull it down.